ansible/playbooks/AdoptOpenJDK_AIX_Playbook/roles/yum/tasks/main.yml

```diff
@@ -87,13 +88,25 @@
     - zsh
   tags: yum
 
-##############################################
-# Additional Tools not available through yum #
-##############################################
-- name: Install yum package support
-  yum: name={{ item }} state=present update_cache=yes
-  with_items:
-    - http://www.oss4aix.org/download/RPMS/cmake/cmake-3.7.2-1.aix6.1.ppc.rpm
+#######################################################
+# Additional Tools not available directly through yum #
+#######################################################
+- name: Confirm cmake prereqs have been installed
+  stat:
+    path: /opt/freeware/etc/rpm/macros.cmake
+  register: cmake_prereqs
```

Comment:
Missing a `yum` tag

Reply:
It may also be ideal to give these specific tasks an additional `cmake` tag

Reply:
I think since this is not introducing a new role it doesn't matter too much, but I'll add it :-)